aladdin.sh

```diff
@@ -123,26 +123,38 @@ function enter_minikube_env() {
 }
 
 function _extract_cluster_config_value() {
+    # Try extracting config from cluster config.json, default config.json, then aladdin config.json
     local value
     value="$1"
-    jq -r ".$value" "$ALADDIN_CONFIG_DIR/$CLUSTER_CODE/config.json"
+    jq -enr "first(inputs | .\"$value\" // empty)" "$ALADDIN_CONFIG_DIR/$CLUSTER_CODE/config.json" \
```

Comment:
you can remove the `-e` then you don't need the || true :)  the -e just makes it error on null but if we don't care about that case we can just have the default behavior of empty output on null